src/arb/Complex.jl

```diff
@@ -695,6 +695,23 @@ function rsqrt(x::ComplexFieldElem, prec::Int = precision(Balls))
    return z
 end
 
+@doc raw"""
+    root(x::ComplexFieldElem, n::Int)
+
+Return the principal $n$-th root of $x$.
+"""
+function root(x::ComplexFieldElem, n::Int, prec::Int = precision(Balls))
+   z = ComplexFieldElem()
+   n == 0 && error("cannot take 0-th root")
+   if n < 0
+     n = -n
+     x = inv(x)
+   end
+   ccall((:acb_root_ui, libarb), Nothing, (Ref{ComplexFieldElem}, Ref{ComplexFieldElem}, UInt, Int), z, x, UInt(n), prec)
```

Comment:
It would be good to have a test case for this. Even a simple `jldoctest` in the docstring would fit the bill...